src/Controllers/CopyPatrol.php

```diff
@@ -298,18 +298,8 @@ private function getUserWhitelist() {
 		if ( $whitelist !== null ) {
 			return $whitelist;
 		}
-		// Get whitelist from the cache if possible.
-		$cacheKey = 'copypatrol_user_whitelist';
-		$cacheItem = $this->cache->getItem( $cacheKey );
-		if ( $cacheItem->isHit() ) {
-			$whitelist = $cacheItem->get( $cacheKey );
-		} else {
-			// It doesn't exist or it expired, so fetch from wiki page.
-			$whitelist = $this->wikiDao->getUserWhitelist();
-			// Store in the cache for 2 hours.
-			$cacheItem->set( $whitelist )->expiresAfter( 2 * 60 * 60 );
-			$this->cache->save( $cacheItem );
-		}
+		// It doesn't exist so fetch from wiki page.
+		$whitelist = $this->wikiDao->getUserWhitelist();
```

Comment:
Let's try using the filesystem cache locally (remove redis vars from .env), and if that works for everyone maybe we could keep the caching functionality.